src/Components/Label/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './index.css';
+
+export default class LabelTitle extends Component {
+    render() {
+
+        const arrowImg = "http://via.placeholder.com/18x22";
```

Comment:
Since this is just for demo purposes, can you remove this from here? You can the placeholder `src` url directly into the `<img/>` tag, I think this is better because as soon as we have the right icon we just need to replace the `url`. What do you think?